docs/third-party-cdn.md

```diff
@@ -0,0 +1,60 @@
+# Third Party CDN Usage
+
+In some situations it may be required to use your own CDN to serve the Altis powered website or service. Below are some examples that may necessitate this:
```

Comment:
Use the expanded form of CDN first before using the acronym eg
```suggestion
In some situations it may be required to use your own Content Delivery Network (CDN) to serve the Altis powered website or service. Below are some examples that may necessitate this:
```